tutorials/how-to-setup-applesilicon-server-with-terraform-ansible/index.mdx

```diff
@@ -0,0 +1,229 @@
+---
+meta:
+    title: Automating Apple Silicon Server Creation with Terraform and Ansible
+    description: Learn how to automate the creation and management of Apple Silicon servers using Terraform and Ansible. This guide provides step-by-step instructions for both tools
+content:
+    h1: Automating Apple Silicon Server Creation with Terraform and Ansible
+    description: Learn how to automate the creation and management of Apple Silicon servers using Terraform and Ansible. This guide provides step-by-step instructions for both tools
+categories:
+    - apple-silicon
+    - terraform
+    - ansible
+tags: apple-silicon terraform ansible
+---
+
+In this tutorial, we’ll guide you through automating the setup and management of Apple Silicon servers using two powerful tools: [Terraform](https://www.terraform.io/) and [Ansible](https://www.ansible.com/). By leveraging these tools, you can streamline infrastructure management, reduce manual configuration, and ensure consistent environments.
+
+<Macro id="requirements" />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+- A valid [API key](/identity-and-access-management/iam/how-to/create-api-keys/)
+
+## Understanding the Tools
+
+### HashiCorp Terraform
+[Terraform](https://www.terraform.io/) is an open-source Infrastructure as Code (IaC) tool that allows you to define and manage infrastructure using declarative configuration files. Terraform enables you to create reproducible and scalable environments while automating the deployment of resources.
+
+### Ansible
+[Ansible](https://www.ansible.com/) is an automation tool that simplifies configuration management, application deployment, and task automation. By using playbooks, Ansible helps automate and streamline processes, ensuring consistency and efficiency in managing complex infrastructure.
+
+## How to create an apple-silicon server
+
+### Using Terraform
+
+1. Download and install [Terraform](https://developer.hashicorp.com/terraform/install)
+
+2. Create a Directory: First, create a directory for your Terraform project. Open your terminal and run:
+
+```shell
+mkdir apple_silicon_server_terraform
+cd apple_silicon_server_terraform
+```
+
+3. Create a Terraform Configuration File: Inside this directory, create a file named resources.tf:
+
+```shell
+touch resources.tf
+```
```

Comment:
````suggestion
    ```shell
    touch resources.tf
    ```
````

Fyi: You have to indent all the code examples; otherwise the numbered steps will be broken and display all as step 1 :(